Windows/Debugger/Debugger_Disasm.cpp

```diff
@@ -32,11 +32,33 @@
 #include <windowsx.h>
 #include <commctrl.h>
 
+// Includes for Trace Logger Mod
+#include <fstream>
+#include <string>
+#include <sstream>
 // How long (max) to wait for Core to pause before clearing temp breakpoints.
 static const int TEMP_BREAKPOINT_WAIT_MS = 100;
 
 static FAR WNDPROC DefGotoEditProc;
 
+// Begin Trace Logger Changes
+static int check = CreateDirectoryA("Trace Logs\\", NULL);
```

Comment:
Most of this should be moved either as a class member to to local scope when not needed elsewhere

Also STEP_INTO etc could use a enum.

Reply:
I actually need to review how to use enum.